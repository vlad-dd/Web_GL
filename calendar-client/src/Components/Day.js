import React from "react";
import Event from "./Event";

const Day = props => {
  return (
    <div className="day">
      <h4>{props.day}</h4>

      <div className="event_container">
        {props.events.map((event, index) => (
          event.day === props.day &&
          <Event 
            key= {event._id}
            event={event.event}
            onEdit={event.onEdit}
            id={event._id}
            color={event.bkgColor}
            dayIndex={props.dayIndex}
            deleteEvent={props.deleteEvent}
            editEvent={props.editEvent}
            updateEvent={props.updateEvent}
            input={props.input}
            updateInput={props.updateInput}
            saveEventChanges={props.saveEventChanges}
          />
        ))}
      </div>
    </div>
  );
};

export default Day;
