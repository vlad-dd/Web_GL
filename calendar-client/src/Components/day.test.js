import React from 'react';
import Day from './Day'
import ReactDOM from 'react-dom';
import TestUtils from 'react-addons-test-utils';




test('the data is correct ', () => {
    expect('Day').toMatchSnapshot();
});



// describe('CalendarTitle', () => {

//     let component;
//     let node; 

//     beforeEach(() => {
//         component = TestUtils.renderIntoDocument(<div><Day title={props.day}/></div>);
//         node = ReactDOM.findDOMNode(component);
//     });

//     it('displays a title supplied', () => {

//         // Verify that its textContent is "Sunday"
//         expect(node.querySelector('div').textContent).toBe(title);

//     })

// })