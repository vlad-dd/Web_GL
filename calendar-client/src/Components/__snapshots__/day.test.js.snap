// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the data is correct  1`] = `"Day"`;

exports[`the login data is correct  1`] = `"Day"`;
